src/main/java/com/box/sdk/SharedLinkAPIConnection.java

```diff
@@ -93,14 +93,50 @@ public boolean getAutoRefresh() {
         return this.wrappedConnection.getAutoRefresh();
     }
 
+    /**
+     * Sets the total maximum number of times an API request will be tried when error responses
+     * are received.
+     * @return the maximum number of request attempts.
+     * @deprecated getMaxRetryAttempts is preferred because it more clearly gets the number
+     * of times a request should be retried after an error response is received.
+     */
+    @Deprecated
     @Override
     public int getMaxRequestAttempts() {
-        return this.wrappedConnection.getMaxRequestAttempts();
+        return this.wrappedConnection.getMaxRetryAttempts();
```

Comment:
```suggestion
        return this.wrappedConnection.getMaxRetryAttempts() + 1;
```

Reply:
Good point, but adding plus one to `getMaxRetryAttempts` is better because `getMaxRequestAttempts` is deprecated, now.